tutorials/00-getting-started.md

```diff
@@ -48,6 +48,8 @@ The default production configuration includes 2 redis instances. You can **great
 
 For development usages, it is helpful to set "linked form and data server" -> "server url" to `""`, so you can use any OpenRosa server with your local Enketo Express.
 
+To prevent SSRF attacks, a `IPFiltering` object is added to the `default-config.json` configuration that prevents private IP addresses from making GET and HEAD requests.
+
```

Comment:
Maybe just make it really easy and tell devs exactly how you recommend they help running Enketo Express locally to develop on, i.e. what setting to tweak in config.json during development.